scripts/opencode-model-scout.sh

```
@@ -0,0 +1,295 @@
+    local -a alive=()
+    local p
+    for p in "${jobs[@]}"; do
+      kill -0 "$p" 2>/dev/null && alive+=("$p")
+    done
+    jobs=("${alive[@]}")
+  done
+}
+
+export -f safe_name non_chat discover_candidates run_pass
+
+ACTIVE_PIDS_MAIN_PID="$$"
```

Shellcheck (0.11.0)

[warning] ACTIVE_PIDS_MAIN_PID appears unused. Verify use (or export if used externally).

(SC2034)